src/authorization/AclOwnershipReader.ts

```diff
@@ -0,0 +1,52 @@
+import { CredentialGroup } from '../authentication/Credentials';
+import type { AccountSettings, AccountStore } from '../identity/interaction/email-password/storage/AccountStore';
+import type { AuxiliaryIdentifierStrategy } from '../ldp/auxiliary/AuxiliaryIdentifierStrategy';
+import type { AclPermission } from '../ldp/permissions/AclPermission';
+import type { PermissionSet } from '../ldp/permissions/Permissions';
+import type { PermissionReaderInput } from './PermissionReader';
+import { PermissionReader } from './PermissionReader';
+
+/**
+ * Allows full access on ACL resources if the request is being made by the owner of the pod containing that resource.
+ */
+export class AclOwnershipReader extends PermissionReader {
```

Comment:
Confusing name though.

`OwnerPermissionReader` or `PodOwnerPermissionReader`?

Reply:
I wanted to have `acl` in the name since this class is only going to work with acl authentication because of the `control` access mode.

Reply:
Yeah but ACL points to the whole WebACL system, not specifically to the WebACL document (which is what you want here).

`ControlPermissionReader`?

But still think the "owner" labels are more descriptive.

Reply:
You could call it premature (name) optimization, but I was just thinking of the future when we also want to give the owner access when using a different auth system than acl. We would need a reader with a different name there.

Reply:
Then I'd personally go for `OwnerPermissionReader`, but your call!